modules/ROOT/nav.adoc

```diff
@@ -1,5 +1,6 @@
 .xref:index.adoc[Mule Runtime]
 * xref:index.adoc[About Mule Runtime]
+* xref:hardware-and-software-requirements.adoc[Hardware and Software Requirements]
```

Comment:
Dev confirmed this is for all Mule Runtime Engines of this version (4.1), so I moved the topic up.

Reply:
@Dukesphere Please identify all the caveats you're aware of and I'll add them and run by tech review again.